tests/unit/executors/indexers/test_binary_indexer.py

```diff
@@ -30,3 +30,111 @@ def validate(req):
         d.ClearField('embedding')
     with f:
         f.search(docs_no_embedding, output_fn=validate, override_doc_id=False)
+
+
+def test_binarypb_update(tmpdir):
+    # required for creating workspace
+    metas = {
```

Comment:
This can be a pytest fixture used for below tests broken into many tests.

Reply:
yes